packages/miniflare/src/plugins/core/inspector-proxy/inspector-proxy-controller.ts

```diff
@@ -20,18 +21,22 @@ export class InspectorProxyController {
 
 	#proxies: InspectorProxy[] = [];
 
-	#server: Server;
+	#server: Promise<Server>;
+
+	#inspectorPort: number | Promise<number>;
 
 	constructor(
-		private userInspectorPort: number,
+		userInspectorPort: number,
 		private log: Log,
 		private workerNamesToProxy: Set<string>
 	) {
+		this.#inspectorPort =
+			userInspectorPort !== 0 ? userInspectorPort : getPort();
```

Comment:
Is this definitely needed? I _think_ we should be able to pass `0` all the way to workerd, and have workerd resolve it?

Reply:
But this doesn't have anything to do with workerd, we create a nodejs server that handles this communication (and proxies things to workerd): https://github.com/cloudflare/workers-sdk/blob/5b91294699196fcc484edfbc2a456163abb5a4f6/packages/miniflare/src/plugins/core/inspector-proxy/inspector-proxy-controller.ts#L58